sanic/sessions.py

```diff
@@ -0,0 +1,303 @@
+# -*- encoding: utf-8 -*-
+
+import hashlib
+
+from itsdangerous import URLSafeTimedSerializer
+from datetime import datetime
+
+import ujson as json
+from sanic.exceptions import BadSignature
+
+
+def total_seconds(td):
+    """Returns the total seconds from a timedelta object.
+    :param timedelta td: the timedelta to be converted in seconds
+    :returns: number of seconds
+    :rtype: int
+    """
+    return td.days * 60 * 60 * 24 + td.seconds
+
+
+class SessionMixin(object):
```

Comment:
This is essentially copy paste from Flask: https://github.com/pallets/flask/blob/master/flask/sessions.py#L22.
This may not have to be a built-in feature of Sanic. 
https://pythonhosted.org/Flask-Session/, I'd like to say you can import the module or make a Sanic one.